serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: llama-2-70b
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: llama-2-70b
+  template:
+    metadata:
+      labels:
+        app: llama-2-70b
+    spec:
+      containers:
+      - name: llama-2-70b
+        image: ghcr.io/huggingface/text-generation-inference:1.0.3
+        resources:
+          limits:
+            nvidia.com/gpu: 2
+        env:
+        - name: MODEL_ID
+          value: meta-llama/Llama-2-70b-chat-hf
+        - name: NUM_SHARD
+          value: "2"
+        - name: PORT 
+          value: "8080"
+        - name: QUANTIZE
```

Comment:
It might be worth adding a short sentence explaining what quantization is and why we are doing it for inference, with a link to more detailed information if the user is interested in learning more.

Reply:
I didn't find any good blogs that I would link to so I rather leave that up to the user to figure out. I did add some comments on why we use quantization and explained it.

Reply:
I think we should at least explain why we selected this value.

Reply:
I added an explanation already and also added explanation on how this setting impacts how much GPU memory is needed